src/controllers/scripting/legacy/controllerscriptenginelegacy.cpp

```diff
@@ -323,7 +324,11 @@ void ControllerScriptEngineLegacy::setScriptFiles(
     m_scriptFiles = scripts;
 
 #ifdef MIXXX_USE_QML
-    setQMLMode(std::any_of(std::execution::par_unseq,
+    setQMLMode(std::any_of(
+#ifndef _LIBCPP_VERSION // Still experimental in Clang's libc++ as of June 2024:
+                        // https://github.com/llvm/llvm-project/issues/65125
+            std::execution::par_unseq,
+#endif
```

Comment:
Judging by the fact that `std::execution` seems very much broken still lets remove it outright. In that case the weird `#pragma push_macro` at the top of the file can also be removed. wdyt?

Reply:
Sounds good to me, wdyt @acolombier?

Reply:
Yep, sounds good to me. 
@Swiftb0y wanted me to put so if he is happy to remove, I'm happy too! :D